src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -1,9 +1,10 @@
-﻿using Dynamo.Graph.Workspaces;
-using Dynamo.Wpf.Extensions;
+﻿using System;
 using System.Collections.ObjectModel;
 using System.ComponentModel;
 using System.Windows.Controls;
 using System.Windows.Media;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Wpf.Extensions;
```

Comment:
Can you update your VS setting to have system usings after custom ones? I think I have updated code style so it is easier for all of us follow that?

Reply:
Yes, changing it. I thought we were placing system directives first.